java/compaction/compaction-rust/src/main/java/sleeper/compaction/rust/RustBridge.java

```diff
@@ -133,6 +133,7 @@ public static class FFICompactionParams extends Struct {
         public final Struct.Boolean override_aws_config = new Struct.Boolean();
         public final Struct.UTF8StringRef aws_region = new Struct.UTF8StringRef();
         public final Struct.UTF8StringRef aws_endpoint = new Struct.UTF8StringRef();
+        public final Struct.Boolean aws_allow_http = new Struct.Boolean();
```

Comment:
Struct definitions including field type and order on the Java side and Rust side MUST match otherwise the field offsets inside the structure will be different.

Reply:
That's fixed, thanks.